### PR TITLE
Start using fluent syntaxing

### PR DESCRIPTION
**Pull request by ImUrX**
For now I only modified parameters that used variables and added plurality and that kind of stuff

### CODE REVIEW

File: gui/public/i18n/en/translation.ftl [hunk 1]
```diff
@@ -135,7 +135,34 @@ body_assignment_menu-manage_trackers = Manage all trackers
...
 ## Tracker assignment menu
-tracker_selection_menu-title = Which tracker to assign to the { $bodyPart }?
+
+# A -translation_key (with a dash in the front) means that it's a label.
+# It can only be used in the translation file, it's nice for reusing names and that kind of stuff.
+# 
+# We are using it here because english doesn't require changing the text in each case but
+# maybe your language does.
+-tracker_selection-part = Which tracker to assign to your
+tracker_selection_menu-title-NONE = { -tracker_selection-part } unassigned
```
Comment (TheDevMinerTV) [inline gui/public/i18n/en/translation.ftl:145]: uhhh: `Which tracker to assign to your unassigned`
wat

Comment (ImUrX) [reply gui/public/i18n/en/translation.ftl:145]: What I want to know is if its even possible to get this query

Comment (ImUrX) [reply gui/public/i18n/en/translation.ftl:145]: because there is no logical way to put this lol

Comment (ImUrX) [reply gui/public/i18n/en/translation.ftl]: Hope you like this one :P

**Comment by ImUrX** [comment]
Will make some people test it now :sob:

**Comment by ImUrX** [comment]
Big PR, what should be reviewed is at least one ``useLocalization()`` hook to see if it's okay how it looks, the ``src/i18n/config.tsx``, maybe see ``scripts/check-missing.js`` (its for checking for missing keys and extra keys), and try the app in every language and see if in the inspector console there is any warning from fluent about a missing key

**Comment by ImUrX** [comment]
Oh and also the english fluent file :p

**Review by Eirenliel — APPROVED** [review]
Please resolve the conflict and rebase to main

**Comment by ImUrX** [comment]
Rebased

**Comment by Eirenliel** [comment]
main also doesn't build, and it looks like fluent error: `[vite]: Rollup failed to resolve import "@fluent/langneg" from "src/i18n/config.tsx".` so manbe you can fix it in this PR, @ImUrX?

**Comment by ImUrX** [comment]
Yeah, the rebase added something I removed, it was weird

**Comment by Eirenliel** [comment]
It was from one of the other merges...

**Comment by macabeus** [comment]
Hey, I apologize for bumping this 2-years old PR, but I'm studying Fluent and found this interesting PR.

Do you recall why you migrated from i18next to fluent, and give more context about the reasoning for that, please?

Thanks!

**Comment by ImUrX** [comment]
i think it was because of better integration of attributes and stuff? And I might have been tempted by the Localization element that fluent-react provides

**Comment by ImUrX** [comment]
yeah... 
image

**Pull request merged**